src/test/scala/com/redislabs/provider/redis/sql/SparkSqlStandaloneSuite.scala

```diff
@@ -0,0 +1,110 @@
+package com.redislabs.provider.redis.sql
+
+import com.redislabs.provider.redis.df.DefaultTestDataset
+import com.redislabs.provider.redis.rdd.{Person, RedisStandaloneSuite}
+import org.apache.spark.sql.redis.RedisFormat
+import org.scalatest.Matchers
+
+/**
+  * @author The Viet Nguyen
+  */
+class SparkSqlStandaloneSuite extends RedisStandaloneSuite with DefaultTestDataset with Matchers {
+
+  test("create temporary table then make regular insertions") {
+    val tableName = Person.generatePersonTableName()
+    // TODO: retrieve table name from tableName instead of path option
+    spark.sql(
+      s"""CREATE TEMPORARY TABLE $tableName (name STRING, age INT, address STRING, salary DOUBLE)
```

Comment:
`CREATE TEMPORARY TABLE` is deprecated, let's use `CREATE TEMPORARY VIEW` instead (sorry if the task description was misleading)

This is printed in the logs
`18/10/01 10:20:35 WARN SparkSqlAstBuilder: CREATE TEMPORARY TABLE ... USING ... is deprecated, please use CREATE TEMPORARY VIEW ... USING ... instead`